package.json

```diff
@@ -51,7 +51,12 @@
   "module": "lib.esm.js",
   "types": "index.d.ts",
   "dependencies": {
+    "@date-io/date-fns": "^1.3.13",
+    "@date-io/moment": "^1.3.13",
     "@material-ui/core": "^4.10.2",
+    "@material-ui/pickers": "^3.2.10",
+    "date-fns": "^2.16.1",
+    "moment": "^2.29.1",
```

Comment:
Again these packages are for date pickers, not time pickers. You'd only need  `KeyboardTimePicker` from `@material-ui/pickers`